src/coreComponents/fileIO/vtk/VTKPolyDataWriterInterface.hpp

```diff
@@ -251,6 +251,31 @@ class VTKPolyDataWriterInterface
   void writeElementFields( vtkSmartPointer< vtkCellData > celldata,
                            ElementRegionBase const & er ) const;
 
+  /*!
+   * @brief Writes all the fields with up to 3 dimensions associated to the elements of \p er if their plotlevel is <= m_plotLevel
```

Comment:
What about scalar (1D) data? Your doc lets me assume it should be OK, but not the name of the function.